packages/perseus/src/widgets/__tests__/radio/focus-ring_test.jsx

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import {render, screen} from "@testing-library/react";
+import * as React from "react";
+
+import "@testing-library/jest-dom"; // Imports custom mathers
+
+import FocusRing from "../../radio/focus-ring.jsx";
+
+describe("choice icon", () => {
+    describe.each([[true], [false]])("multipleSelect: %s", (multipleSelect) => {
+        it("renders with the correct border radius", () => {
+            let expectedRadius;
+            if (multipleSelect) {
+                expectedRadius = "5px";
+            } else {
+                expectedRadius = "50%";
+            }
```

Comment:
**style nit (non-blocking):** a ternary would allow `expectedRadius` to be `const` while still being readable.